.github/workflows/integration_tests.yml

```
@@ -111,6 +108,13 @@ jobs:
       run: cd tests/playwright && npm ci && npm run install-deps
     - name: Run Playwright tests
       run: cd tests/playwright && npm run integration-tests
+    - uses: actions/upload-artifact@v4
+      if: ${{ !cancelled() }}
+      with:
+        name: test-results-${{ github.event.inputs.git_sha }}-${{ github.run_id }}
```

Or if it gets too long, it can be shortened to just:

```suggestion
        name: test-results-${{ github.run_id }}
```

(but I left the triggering sha from cron in place too, if folks want to cross-reference the versions the results came from later… but it's the run_id making the artifact unique really.)